keras_cv/models/object_detection/yolo_v8/yolo_v8_label_encoder.py

```diff
@@ -297,7 +297,7 @@ def get_box_metrics(
 
         gt_boxes = tf.repeat(tf.expand_dims(gt_bboxes, axis=2), na, axis=2)
 
-        iou = tf.squeeze(bbox_iou(gt_boxes, pd_boxes), axis=-1)
+        iou = CIoULoss(gt_boxes, pd_boxes)
```

Comment:
It feels very strange to me to be calling a loss here -- this should be something like `bounding_box.compute_ciou` to match `bounding_box.compute_iou`.

How about we move the `ciou` computation helper to `bounding_box/iou.py`, export it as `bounding_box.compute_ciou`, and use it both here and in the loss

(Sorry I should have thought of this during your last PR)

Reply:
No problem I will add compute_ciou in iou.py :)